src/App.tsx

```diff
@@ -2,108 +2,116 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { Post } from './components/Types/Post';
+import { PostList } from './components/PostList';
+
+const arrayPosts: Post[] = posts.map(post => ({
+  ...post,
+  user: users.find((user) => user.id === post.userId) || null,
+  comments: comments.filter((comment) => comment.postId === post.id),
```

Comment:
you can do like this ( UP TO YOU )
```suggestion
  user: users.find(({ id }) => id === post.userId) || null,
  comments: comments.filter(({ postId }) => comment.postId === post.id),
```